changelog.json

```
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "2.9.12-beta3": [
+      "[Fixed] Escape Special Characters when adding a file to .gitignore - #10253. Thanks @uttiya10!",
+      "[Fixed] The markdown parser correctly set's it's height relative to it's content - #14196",
```

```suggestion
      "[Fixed] Escape special characters when adding a file to .gitignore - #10253. Thanks @uttiya10!",
```

AFAIK #14196 is not user facing, right? We're not using `SandboxedMarkdown` anywhere in the app in prod or beta.

---

Yep - You are right. I guess I spend so much time in it I forgot. 😸